gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/infra/event/EventQueryServiceImplTest.java

```diff
@@ -90,7 +90,7 @@ void should_query_for_events() {
             verify(eventRepository).search(queryCaptor.capture(), pageableCaptor.capture());
             assertThat(queryCaptor.getValue())
                 .satisfies(criteria -> {
-                    assertThat(criteria.getEnvironments()).isEqualTo(List.of("environment-id"));
+                    assertThat(criteria.getEnvironments()).isEqualTo(Set.of("environment-id"));
```

Comment:
Are you sure that changing the list to a set won't cause a flaky test, since the order is no longer guaranteed?

Reply:
It does a `containsAll` in the end and I didn't notice any issue regarding this change when executing all the tests so I guess that's fine 👍